doc/maintenance.md

```diff
@@ -24,6 +24,20 @@ instead.
   branch are in the `dev` folder. Pages for each release are in their own folders.
   **Automatically updated by GitHub Actions** so you shouldn't have to make commits here.
 
+## Managing GitHub issues
+
+A few guidelines for managing GitHub issues:
+
+* Assign [labels](https://github.com/GenericMappingTools/pygmt/labels) and the expected
+  [milestone](https://github.com/GenericMappingTools/pygmt/milestones) to issues as
+  appropriate.
+* When people request to work on an open issue, either assign the issue to that person
+  and post a comment about the assignment or explain why you are not assigning the 
+  issue to them and, if possible, recommend other issues for them to work on.
+* People with write access should self-assign issues and/or comment on the issues that
+  they will address.
+* For upstream bugs, close the issue after an upstream release fixes the bug. If
+  possible, also post a comment when an upstream PR is merged that fixes the problem.
```

Comment:
Just wanted to double check. There are still some upstream issues we didn't close because we were waiting for a GMT version bump (e.g. #392) and/or we need to add tests in PyGMT itself (e.g. #732). Should we clarify the wording a bit, as it reads as though we can close the issue directly once it is fixed upstream (without waiting for a GMT release).

Reply:
Hm I tried to write it so that it's clear that a release is needed. I am glad to revise if you have wording that would make this more clear. Are tests always needed in PyGMT for upstream bugs or is this a case-by-case basis?

Reply:
> Hm I tried to write it so that it's clear that a release is needed. I am glad to revise if you have wording that would make this more clear.

Sorry, should be more clear. I was meaning to ask if we should close the upstream issue when A) GMT 6.2.0 is released, or when B) PyGMT sets the minimum required GMT version to GMT 6.2.0 (or any other version e.g. 6.2.0rc2 where the bugfix was made).

> Are tests always needed in PyGMT for upstream bugs or is this a case-by-case basis?

It's on a case by case basis I think. There were a few nasty `grd*` bugs that we have tests for, while some minor ones might not need tests. Should we mention it here that tests may be required before closing the issue? Or just keep the sentence short as is?

Reply:
> > Hm I tried to write it so that it's clear that a release is needed. I am glad to revise if you have wording that would make this more clear.
> 
> Sorry, should be more clear. I was meaning to ask if we should close the upstream issue when A) GMT 6.2.0 is released, or when B) PyGMT sets the minimum required GMT version to GMT 6.2.0 (or any other version e.g. 6.2.0rc2 where the bugfix was made).
> 
Oh, thanks for the explanation. I would prefer A to have fewer open issues but am OK with B if that is what others prefer.
> > Are tests always needed in PyGMT for upstream bugs or is this a case-by-case basis?
> 
> It's on a case by case basis I think. There were a few nasty `grd*` bugs that we have tests for, while some minor ones might not need tests. Should we mention it here that tests may be required before closing the issue? Or just keep the sentence short as is?

I can add this to the point.

Reply:
Let's go with A (close upstream issue with new GMT release) to keep fewer issues open. Here's a suggested change, feel free to tweak it before committing:

```suggestion
* For upstream bugs, close the issue after an upstream release fixes the bug. If
  possible, post a comment when an upstream PR is merged that fixes the problem, and
  consider adding a regression test for serious bugs.
```